src/main/scala/org/broadinstitute/dsde/workbench/sam/service/ResourceService.scala

```diff
@@ -916,32 +971,12 @@ class ResourceService(
   def getResourceParent(resourceId: FullyQualifiedResourceId, samRequestContext: SamRequestContext): IO[Option[FullyQualifiedResourceId]] =
     accessPolicyDAO.getResourceParent(resourceId, samRequestContext)
 
-  /** In this iteration of hierarchical resources, we do not allow child resources to be in an auth domain because it would introduce additional complications
-    * when keeping Sam policies with their Google Groups. For more details, see
-    * https://docs.google.com/document/d/10qGxsV9BeM6-N_Zk27_JIayE509B8LUQBGiGrqB0taY/edit#heading=h.dxz6xjtnz9la
```

Comment:
Is this still a concern?

Reply:
It is no longer a concern. I am not sure why I though it was in the first place. I was probably thinking about how we might sync inherited policies to google, but thinking about it now, those policies don't actually exist so there is no way to sync them and there is no reason to since the membership will always be the same as the source policy. So if you need to sync a group in order to add it google IAM somewhere, just use the source policy.